rcl_lifecycle/src/com_interface.hxx

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COM_INTERFACE_HXX_
+#define COM_INTERFACE_HXX_
```

Comment:
`test_communication` does lint its files (https://github.com/ros2/system_tests/blob/8074a4128b785c6c2376ec432e94d8bc5e85ff04/test_communication/CMakeLists.txt#L16) and uses a local header too (https://github.com/ros2/system_tests/blob/8074a4128b785c6c2376ec432e94d8bc5e85ff04/test_communication/test/message_fixtures.hpp).